### PR TITLE
Service specific manifests

### LINKED ISSUE #1134: Point at namespaced manifest.json
Blocked on https://github.com/BBC-News/simorgh/issues/1133

**Is your feature request related to a problem? Please describe.**
We should start pointing at our namespaced manifest.json
Currently we point at `<link rel="manifest" href="/manifest.json">`

**Describe the solution you'd like**
Start using the new namespaced manifest.json file

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/simorgh/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by sareh**
Resolves #1134 #1135

**Overall change:** Allows service-specific manifest files to be served by the application.
e.g. at http://localhost:7080/news/articles/manifest.json & http://localhost:7080/persian/articles/manifest.json

**Code changes:**

- Moved News Manifest file & associated icons into `public/news/`
- Added Persian manifest file & icons to `public/persian`
- Moved Manifest `<link>` tag to Articles container, to be able to use the `service` value. 
- Added new route to handle a request to `/:service/articles/manifest.json` to serve file from location `public/:service/manifest.json`

**Test instructions **
`npm run build && npm run start` to ensure you build the static files from `public` into `build/public` and then serve these. 

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: public/news/manifest.json [hunk 1]
```diff
@@ -8,42 +8,42 @@
   "theme_color": "#b80000",
   "icons": [
     {
-      "src": "images/icons/icon-72x72.png",
+      "src": "news/images/icons/icon-72x72.png",
```
Comment (dr3) [inline public/news/manifest.json]: I think we probably need to make these absolute now that we route via bbc.com/co.uk, Happy to hard code them on `https://news.files.bbci.co.uk/` for now though

**Review by dr3 — CHANGES_REQUESTED** [review]
Looks good, just a small thing :)

File: public/news/manifest.json (hunk 1 above)
Comment (sareh) [reply public/news/manifest.json]: Done.

File: src/app/containers/Article/__snapshots__/index.test.jsx.snap [hunk 2]
```diff
@@ -149,6 +158,15 @@ exports[`ArticleContainer Component non-200 status code should render correctly
     <PlatformContextProvider
       platform="canonical"
     >
+      <HelmetWrapper
+        defer={true}
+        encodeSpecialCharacters={true}
+      >
+        <link
+          href="/news/articles/manifest.json"
```
Comment (pjlee11) [inline src/app/containers/Article/__snapshots__/index.test.jsx.snap:288]: Would be good to have a snapshot test for persian

File: src/server/index.jsx [hunk 3]
```diff
@@ -87,6 +88,16 @@ server
       }
     });
   })
+  .get(manifestRegexPath, async ({ params }, res) => {
+    const { service } = params;
+    const manifestPath = `${__dirname}/public/${service}/manifest.json`;
```
Comment (pjlee11) [inline src/server/index.jsx:93]:
```suggestion
    const manifestPath = `./${publicDirectory}/${service}/manifest.json`;
```

Does this need to be the absolute path that `__dirname`returns or could it just be `./build/public`?

**Review by dr3 — DISMISSED** [review]

File: src/server/index.jsx (hunk 3 above)
Comment (sareh) [reply src/server/index.jsx:93]: `res.sendFile` wants an absolute path and not a relative one. 
See the error when using a relative path: `(node:46712) UnhandledPromiseRejectionWarning: TypeError: path must be absolute or specify root to res.sendFile`

**Review by dr3 — APPROVED** [review]

**Review by pjlee11 — APPROVED** [review]
LGTM 👍

**Pull request merged**